contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -606,11 +696,11 @@ contract NodeOperatorsRegistry is INodeOperatorsRegistry, IsContract, AragonApp
         }
 
         operators[_operator_id].totalSigningKeys = operators[_operator_id].totalSigningKeys.add(to64(_quantity));
+
+        _updateTotalAvailableKeysCount();
     }
 
-    function _removeSigningKey(uint256 _operator_id, uint256 _index) internal
-        operatorExists(_operator_id)
-    {
+    function _removeSigningKey(uint256 _operator_id, uint256 _index) internal operatorExists(_operator_id) {
```

Comment:
make it much simpler (to resemble trim mechanics not erasing storage explicitly, only move counters)